### DISCUSSION TITLE
ENGIE

### DISCUSSION THREAD

**Discussion by Victoravocataix** [JC144/EDF_Simulateur_Prix#110] [General]
Pourquoi ne pas ajouter l'offre Elec Référence 2 ans de ENGIE qui propose un kwh 0,14857 € ? Elec référence 1 le propose même à 0,13951 € !

**Comment by JC144** [comment]
Bonjour,

Vous pouvez ajouter les tarifs de ENGIE si vous le souhaitez, la procédure est indiquée ici : https://github.com/JC144/EDF_Simulateur_Prix/wiki/Ajout-de-tarifs

Je serai ravi de les valider ensuite.

J'ai eu peu de temps à consacrer au projet ces derniers temps et la maintenance des tarifs est assez chronophage, c'est pour ça que je me concentre particulièrement sur ceux d'EDF.
